walk-packages-for-changelog.py

```diff
@@ -391,13 +393,19 @@ def main():
             continue
 
         header = ''
-        if origin_url and origin_url.startswith('https://github.com'):
+        from_github = origin_url.startswith('https://github.com')
+        from_gitlab = origin_url.startswith('https://gitlab.com')
+        if origin_url and (from_github or from_gitlab):
             if origin_url.endswith('.git'):
                 origin_url = origin_url[:-4]
             url_path = '/'
             if len(package.dirpath) != len(repo_path):
```

Comment:
I guess I could move `os.path.abspath(...)` to here, e.g.
```suggestion
            package.dirpath = os.path.abspath(package.dirpath)
            if len(package.dirpath) != len(repo_path):
```